integration/nwo/fabric/network/resolver.go

```diff
@@ -57,12 +56,15 @@ func (n *Network) GenerateResolverMap() {
 				path = n.PeerLocalMSPIdentityCert(peer)
 			}
 		} else {
-			addresses = map[api.PortName]string{
-				//ViewPort:   fmt.Sprintf("127.0.0.1:%d", n.Context.PortsByPeerID(n.Prefix, peer.ID())[ListenPort]),
-				ListenPort: fmt.Sprintf("127.0.0.1:%d", n.Context.PortsByPeerID(n.Prefix, peer.ID())[ListenPort]),
-				//P2PPort:    fmt.Sprintf("127.0.0.1:%d", n.Context.PortsByPeerID(n.Prefix, peer.ID())[P2PPort]),
-			}
-			path = n.PeerLocalMSPIdentityCert(peer)
+			//addresses = map[api.PortName]string{
+			//	//ViewPort:   fmt.Sprintf("127.0.0.1:%d", n.Context.PortsByPeerID(n.Prefix, peer.ID())[ListenPort]),
+			//	ListenPort: fmt.Sprintf("127.0.0.1:%d", n.Context.PortsByPeerID(n.Prefix, peer.ID())[ListenPort]),
+			//	//P2PPort:    fmt.Sprintf("127.0.0.1:%d", n.Context.PortsByPeerID(n.Prefix, peer.ID())[P2PPort]),
+			//}
+			//path = n.PeerLocalMSPIdentityCert(peer)
+
```

Comment:
can this be removed?

Reply:
actually no because the FPC Echo fails. So, I will restore it for now until I found a way to discover the info of the peer endpoints hosting the enclave.